src/js/letters/utils/helpers.jsx

```diff
@@ -255,7 +255,8 @@ export function getBenefitOptionText(option, value, isVeteran, awardEffectiveDat
     valueString = value;
   }
 
-  const isAvailable = value && value !== AVAILABILITY_STATUSES.unavailable;
```

Comment:
@kreek can you confirm that the `/beneficiary` endpoint doesn't return a value of `'unavailable'` for any of the `benefitInformation` options?

Reply:
@markgreenburg the data services entry for letters is copied from their swagger (which I'm getting a 403 for on the VPN right now 😐) https://github.com/department-of-veterans-affairs/vets.gov-team/tree/master/Data/Data-Services/EVSS/Letters#get-lettersv1letterbeneficiary and it should either return the value or be `null` (but not `'unavailable'`)

Reply:
Thanks! Looks like taking out this extra condition is the way to go. @cvalarida - do you see any problems with this update that I might be missing?